README.md

```diff
@@ -68,6 +68,11 @@ Set up (the latest version of) [HAProxy](http://www.haproxy.org/) in Ubuntu syst
 * `haproxy_defaults_compression`: [optional]: Compression declarations
 * `haproxy_defaults_compression.{}.name`: [required]: The compression name (e.g. `algo`, `type`, `offload`)
 * `haproxy_defaults_compression.{}.value`: [required]: The compression value, (e.g. if name = algo : one of this values `identity`, `gzip`, `deflate`, `raw-deflate` / if name = type : list of mime type separated by space for example `text/html text/plain text/css` / if name = `offload` value is empty)
+  
+* `haproxy_resolvers_label`: [default: `[]`]: The global name of your dns resolver
```

Comment:
Can you remove this variable and make `haproxy_resolvers_list` look like `haproxy_userlists`?

Reply:
`haproxy_resolverslist`